python/simulation.py

```diff
@@ -859,6 +864,155 @@ def amplitude(self, point, component):
         return mp.eigenmode_amplitude(self.swigobj, swig_point, component)
 
 
+class Pade:
+    """
+    Pade is implemented as a class with a [`__call__`](#Pade.__call__) method,
```

Comment:
Would be more accurate to write Pade as Padé in the docstrings.

Reply:
https://github.com/NanoComp/meep/pull/2440/commits/6dd0939cac5f12cd11ece3b659ddd524f121f41e